### PR TITLE
Fix broken migrations

### PR DESCRIPTION
**Pull request by cammellos**
migrations was noop, it should be fixed now

### CODE REVIEW

**Review by caybro — COMMENTED** [review]
does not fix the issue for me on status-desktop/master

**Review by jrainville — APPROVED** [review]
I tested both from before the faulty migration to this fix and by applying the faulty migration and both worked with this.

File: protocol/migrations/sqlite/1674210659_add_contact_request_local_clock.up.sql [hunk 1]
```diff
@@ -3,13 +3,16 @@ ALTER TABLE contacts ADD COLUMN contact_request_local_clock INT;
 ALTER TABLE contacts ADD COLUMN contact_request_remote_clock INT;
 ALTER TABLE contacts ADD COLUMN contact_request_remote_state INT;
 
+-- Broken migration, leaving for posterity and eternal embarrassment
```
Comment (richard-ramos) [inline protocol/migrations/sqlite/1674210659_add_contact_request_local_clock.up.sql:6]: :medal_military:

**Review by richard-ramos — APPROVED** [review]

**Review by caybro — APPROVED** [review]

**Comment by caybro** [comment]
My bad, after a forced rebuild seems to work fine, thanks for the quick fix!

**Comment by cammellos** [comment]
> My bad, after a forced rebuild seems to work fine, thanks for the quick fix!

Thank you for testing, and apologies for the mix up

**Pull request merged**